feat: add `@stdlib/blas/base/srotg` (#806)
stdlib-js/stdlib: lib/node_modules/@stdlib/blas/base/srotg/docs/repl.txt

{{alias}}( a, b )
    Constructs a Givens rotation from two single-precision 
    floating-point numbers.

    Parameters
    ----------
    a: float
        Rotational elimination parameter.
    
    b: float
        Rotational elimination parameter.

    Returns
    -------
    out: Float32Array
        Computed values.

    Examples
    --------
    // Standard usage:
    > var out = {{alias}}( 0.0, 2.0 )
    <Float32Array>[ 2.0, 1.0, 0.0, 1.0 ]

>     See Also

Reviewer: We should also document the `assign` API, as it is part of the public interface.

{{alias}}( a, b )
    Constructs a Givens rotation from two single-precision 
    floating-point numbers.

    Parameters
    ----------
    a: float
        Rotational elimination parameter.
    
    b: float
        Rotational elimination parameter.

    Returns
    -------
    out: Float32Array
        Computed values.

    Examples
    --------

>     // Standard usage:
>     > var out = {{alias}}( 0.0, 2.0 )

Reviewer: ```suggestion
    > var out = {{alias}}( 0.0, 2.0 )
```

    <Float32Array>[ 2.0, 1.0, 0.0, 1.0 ]

{{alias}}.assign( a, b, out, stride, offset )
    Constructs a Givens rotation from two single-precision 

>     floating-point numbers and assigns the results to a given
>     output array.

Reviewer: ```suggestion
    floating-point numbers and assigns the results to an output
    array.
```

You may need to rewrap.
